Monika After Story/game/script-holidays.rpy

```diff
@@ -18,13 +18,84 @@ define mas_one_hour = datetime.timedelta(seconds=3600)
 define mas_three_hour = datetime.timedelta(seconds=3*3600)
 
 init -1 python:
+    def mas_wasFirstValueIn(_verify, year, *keys):
```

Comment:
Not now, but at some point, can we change that to `mas_histXYZ()`? None of our other functions have the first letter capitalized like that.

Reply:
there are other functions that do that (shields/hotkeys). I like using `mas_<capital>` for related API functions.

A lot of the other functions didn't follow that because of several reasons like:
1. not originally planning on making a bunch of related API functions
2. the function names read better like ` mas_isSStoMN` than `mas_STisSStoMN`